components/core/SearchBox.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+const PLACEHOLDER = 'API Search';
+
+export const SearchBox = ({ onSearch }) => {
+
+  const [searchText, setSearchText] = useState('');
```

Comment:
The searchbox seems to be small in width. can we style it to be more larger but responsive?